document how to provide and use mulitple VEE Ports (#531)
MicroEJ/docs: SDK6UserGuide/selectVeePort.rst

There are 4 different ways to provide a VEE Port in the build file of the project:

- Set the build property ``veePortFiles`` in the ``microej`` configuration block to the path of a VEE Port file (``.zip``, ``.jpf``, or ``.vde``)::

   microej {

>      veePortFiles = listOf("C:\\path\\to\\my\\veePort\\file.zip")

Reviewer: The section looks good, except that it is weird to use `listOf` here without explanation. Either we merge the sections (we directly explain that we can define one or more VEE Ports), or we add a sentence on the usage of `listOf` with a redirection to the next section. WDYT?

Author: I added the sentence :
```
The ``veePortFiles`` property is defined as a list in order to provide multiple VEE Port files if it is needed.
See the :ref:`Select multiple VEE Ports <sdk_6_select_multiple_vee_ports>` section for more information.
```
Idem for the ``veePortDirs`` property. I think it is better to have a dedicated section for the multiple VEE Ports.

      
      Refer to `the Gradle documentation <https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.dsl.DependencyHandler.html>`__ 
      to learn all the options to select dependencies.

- Copy a VEE port archive file to the ``dropins`` folder. The default dropins folder location is ``[module_project_dir]/dropins``. It can be changed using the build property ``veePortDropinsDir``::

   microej {
      veePortDropinsDir = "C:\\path\\to\\dropins"
   }

At least 1 of these 4 ways is required to build an Application with a VEE Port.

.. _sdk_6_select_multiple_vee_ports:

Select multiple VEE Ports
-------------------------

> It is possible to provide multiple VEE Ports to a project using one or more of 
> the :ref:`existing ways <sdk_6_select_veeport>` to provide a VEE Port. Each VEE Port will be used by the project.
> 
> For example, you can set the ``veePortDirs`` to the path of a ``source`` folder of a VEE Port, 
> declare 2 dependencies with the ``microejVeePort`` conf and copy an archive file to the ``dropins`` folder.

Reviewer: ```suggestion
If multiple :ref:`VEE Ports are defined <sdk_6_select_veeport>` for a project, they are all used.
For example, you can set the ``veePortDirs`` to the path of a ``source`` folder of a VEE Port, 
declare 2 dependencies with the ``microejVeePort`` configuration and copy an archive file to the ``dropins`` folder.
```

To add several VEE Ports using the ``veePortFiles`` or the ``veePortDirs`` property, the paths must be separated by a comma. 

> For example ::

Reviewer: ```suggestion
For example::
```

   microej {
     veePortFiles = listOf("C:\\path\\to\\my\\veePort1\\file.zip","C:\\path\\to\\my\\veePort2\\file.zip")
   } 

.. _sdk_6_vee_port_unique_name:

VEE Port unique name
--------------------

> A VEE Port that has been provided has a unique name ``{name}-{toolchain}-{version}``, 

Reviewer: ```suggestion
VEE Ports are referenced by a unique name following the pattern ``{name}-{toolchain}-{version}``, 
```

where ``name``, ``toolchain`` and ``version`` values are the ones defined in the ``release.properties`` file of the VEE Port. 
For example : `Platform-CM7hardfp_GCC48-0.1.0`

> If multiple VEE Ports have been provided, this name can be used to select the VEE Port on which 

Reviewer: ```suggestion
If multiple VEE Ports are defined, this name can be used to select the VEE Port on which 
```
